src/crimson/os/store_context.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "common/perf_counters_collection.h"
+#include "ConfigObs.h"
+
+class StoreContext {
+  ConfigObs *cobs;
+  PerfCountersCollection *perf_counters_collection;
+public:
+  StoreContext(): cobs(NULL),perf_counters_collection(NULL) {
```

Comment:
initialize member variable in-class, and please use `nullptr` if possible.

Reply:
sure.